todo-app/src/components/TaskItem.js

```diff
@@ -0,0 +1,66 @@
+import React, { Component } from "react";
+import "../style.css"
+import PropTypes from "prop-types"
+import Trash from "react-icons/lib/fa/trash"
+import Edit from "react-icons/lib/fa/edit"
+
+
+export default class TaskItem extends Component {
+    static propTypes = {
+        id: PropTypes.number,
+        description: PropTypes.string,
+        deadLine: PropTypes.string,
+        done: PropTypes.bool,
+        creatingDate: PropTypes.string
+    }
+    
+    state = {
```

Comment:
This state needs to be lifted. I see id, description, deadLine, done, and creatingDate are all coming as props. The `done` value should definitely be a prop as well, with a change function being passed via props too.